Private/Get-ConfigurationEvents.ps1

```diff
@@ -4,26 +4,18 @@ function Get-CongfigurationEvents {
         $Sections
     )
     $EventFiles = New-ArrayList
-    if ($Sections.Directories.Use) {
+    if ($Sections.ContainsKey("Directories")) {
```

Comment:
Right now the code to use it looks like this. Use is used to disable/enable when required without uncommenting things in configuration. Without it, it doesn't make much sense right? Because you should also check for existance of MyEvents in configuration?

       ArchiveProcessing = @{
            Use         = $false
            Directories = [ordered] @{
                Use      = $false
                MyEvents = 'E:\EventLogs' #
                #MyOtherEvent = 'C:\MyEvent1'
            }
            Files       = [ordered] @{
                Use = $false
                #File1 = 'E:\EventLogs\Archive-Security-2018-09-14-22-13-07-710.evtx'
            }
        }

Reply:
I decided to simplify the configuration a bit:
```
        ArchiveProcessing = @{
            Use         = $true
            Directories = [ordered] @{
                MyEvents     = 'C:\MyEvents' #
                MyOtherEvent = 'C:\MyEvent1'
            }
            Files       = [ordered] @{
                File1 = 'C:\MyEvents\Archive-Security-2018-09-14-22-13-07-710.evtx'
            }
        }
```
The exclusion of an additional configuration key allowed simplifying it in my opinion. In this case, it became possible not to set the configuration of the Directory or File.

Reply:
Ok, I guess simplification is always good. My initial goal was to have ability to define some stuff and simply not use it. Like use Directories, define Files but not use them if not needed. But maybe I'm over complicating things.

Reply:
You can continue to use this strategy simply by commenting out directory paths.